### PR TITLE
Added metrostation. Update house screenshot

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

File: docs/learning/tutorials/create-test-imodel-sample.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ The quickest way to get access to a cloud hosted iModel is by creating an iModel
   - Any other users will also have to have a Bentley account
 - Click Submit
 
+### Metrostation Sample
+
+A metrostation located in Philadelphia. Contains platform, rails, and subway cars.
```
Comment (pmconne) [inline docs/learning/tutorials/create-test-imodel-sample.md:15]: Did you consider turning off the map or making it mostly transparent? The majority of the model including the platform, rails, and subway cars described here is hidden underground.

Comment (williamkbentley) [reply docs/learning/tutorials/create-test-imodel-sample.md:15]: When a user copies the iModel, it doesn't copy the groundBias, so it ends up looking like this:
image

Which isn't great either, but at least the geometry easily visible.

Comment (pmconne) [reply docs/learning/tutorials/create-test-imodel-sample.md:15]: Frankly I'd find a screenshot from inside the station showing the cars, platforms, etc more illustrative/interesting.

Comment (williamkbentley) [reply docs/learning/tutorials/create-test-imodel-sample.md:15]: Good idea. Also, looks like the screenshot got a bit pixelated
https://github.com/imodeljs/imodeljs/blob/423557d49e232ec11d203b60d5e31f89926f4260/docs/learning/tutorials/images/metrostation.PNG

**Review by pmconne — APPROVED** [review]

**Pull request merged**